csrankings-e.csv

```diff
@@ -411,13 +413,14 @@ Eran Omri,Ariel University,http://www.ariel.ac.il/Projects/trp/GeneralInformatio
 Eran Segal,Weizmann Institute of Science,https://www.weizmann.ac.il/pages/search/people?language=english&single=1&person_id=30021,oAD8PrkAAAAJ
 Eran Tromer,Tel Aviv University,http://www.tau.ac.il/~tromer,Rjywx9wAAAAJ
 Eran Yahav,Technion,http://www.cs.technion.ac.il/~yahave,grAfX0MAAAAJ
+Eray Tuzun,Bilkent University,http://www.cs.bilkent.edu.tr/~eraytuzun,D4jovncAAAAJ
+Eray Tüzün,Bilkent University,http://www.cs.bilkent.edu.tr/~eraytuzun,D4jovncAAAAJ
 Erchin Serpedin,Texas A&M at Qatar,https://www.qatar.tamu.edu/programs/ecen/faculty-and-staff/duplicate-of-dr.-mohamed-nounou,THi_ypMAAAAJ
 Erdinç Öztürk,Sabancı University,http://people.sabanciuniv.edu/erdinco,xHYqay0AAAAJ
 Erez Karpas,Technion,https://karpase.net.technion.ac.il,Dya6lhgAAAAJ
 Erez Petrank,Technion,http://www.cs.technion.ac.il/~erez,ME7VLY8AAAAJ
 Erez Zadok,Stony Brook University,http://www3.cs.stonybrook.edu/~ezk,ZG4V_jYAAAAJ
 Ergun Akleman,Texas A&M University,http://www.viz.tamu.edu/faculty/ergun,T5-M6YMAAAAJ
-Erhan Mengusoglu,Hacettepe University,http://cs.hacettepe.edu.tr/files/erhan_mengusoglu_CVa.pdf,CRHfLuoAAAAJ

```

Comment:
Apparently moved to industry, please update `old/industry.csv`.

Reply:
updated